frontend/src/component/project/Project/ProjectFeatureToggles/RowSelectCell/RowSelectCell.tsx

```diff
@@ -7,24 +7,21 @@ interface IRowSelectCellProps {
     onChange: (_?: unknown) => void;
     checked: boolean;
     title: string;
-    noPadding?: boolean;
```

Comment:
this was probably a hack and I'n removing it now because I fixed the underlying issue with extra padding